tests/cloudpickle_test.py

```diff
@@ -893,6 +888,21 @@ def test_builtin_classmethod(self):
         assert depickled_bound_meth(arg) == bound_clsmethod(arg)
         assert depickled_unbound_meth(arg) == unbound_clsmethod(arg)
 
+    @pytest.mark.skipif(
+        sys.version_info >= (3, 10, 8),
```

Comment:
This issue is `CPython` specific. So we should disable this test only for `CPython` implementations of Python.
```suggestion
        (
            sys.version_info >= (3, 10, 8) and
            platform.python_implementation() == 'CPython'
        ),

```